src/App.tsx

```diff
@@ -1,12 +1,71 @@
 import React from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './components/GoodsList';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+type State = {
+  goods: Good[];
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    goods: [],
+  };
+
+  loadAllgoods = async () => {
+    const goods = await getAll();
+
+    this.setState({ goods });
+  };
+
+  load5First = async () => {
+    const goods = await get5First();
+
+    this.setState({ goods });
+  };
+
+  loadRedGoods = async () => {
+    const goods = await getRedGoods();
+
+    this.setState({ goods });
+  };
```

Comment:
you can make one common function for loading goods and pass the needed function as an argument when clicking a button